From a01ca286f9b9ca135e000da269d393e1438bd488 Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Mon, 5 Dec 2011 13:54:19 +0800
Subject: [PATCH 0684/2463] ENGR00169388-2 MX6Q/UART : fill DMA fields for the
 platformdata of mx6q

fill the DMA RX/TX fields for mx6q board.

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_arm2.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_arm2.c b/arch/arm/mach-mx6/board-mx6q_arm2.c
index 1ad09f8..f61719c 100644
--- a/arch/arm/mach-mx6/board-mx6q_arm2.c
+++ b/arch/arm/mach-mx6/board-mx6q_arm2.c
@@ -481,6 +481,8 @@ static const struct anatop_thermal_platform_data
 
 static const struct imxuart_platform_data mx6q_uart1_data __initconst = {
 	.flags = IMXUART_HAVE_RTSCTS | IMXUART_USE_DCEDTE | IMXUART_SDMA,
+	.dma_req_rx = MX6Q_DMA_REQ_UART2_RX,
+	.dma_req_tx = MX6Q_DMA_REQ_UART2_TX,
 };
 
 static inline void mx6q_arm2_init_uart(void)
-- 
1.7.7.4

